go/nbs/store.go

```diff
@@ -200,6 +200,32 @@ func (nbs *NomsBlockStore) GetMany(hashes []hash.Hash) []chunks.Chunk {
 	return resp
 }
 
+func (nbs *NomsBlockStore) CalcReads(hashes []hash.Hash) (reads int, split bool) {
+	reqs := make([]getRecord, len(hashes))
```

Comment:
make a utility for this which is used here & GetMany() to produce this slice of getRecords from hash requests?